config/custom-environment-variables.json

```diff
@@ -100,11 +100,15 @@
       "os-postcode-lookup-api-key": "POSTCODE_LOOKUP_API_KEY",
       "encryptionKey": "COOKIE_ENCRYPTION_KEY",
       "AppInsightsInstrumentationKey":"APPINSIGHTS_INSTRUMENTATIONKEY",
+      "launchDarkly-sdk-key": "LD_SDK_KEY",
```

Comment:
Is 'LD_SDK_KEY' the local environment key? Maybe it could be named LOCAL_LD_SDK_KEY?